packages/orbit-components/src/BadgeList/BadgeListItem/types.d.ts

```diff
@@ -5,9 +5,12 @@ import type * as React from "react";
 import type * as Common from "../../common/types";
 
 export type Type = "neutral" | "info" | "success" | "warning" | "critical";
+export type Size = "small" | "normal";
+
 export interface Props extends Common.Globals {
   readonly children: React.ReactNode;
   readonly type?: Type;
+  readonly size?: Size;
```

Comment:
add the prop also to `index.js.flow` 🙏🏻

Reply:
For some reason I thought the flow files were being generated automatically 🤔

Reply:
we left that idea because of slow flowgen, it takes him a lot to generate files. We use it only on some root package files, like `index.ts` and others